File: Solutions/Java/Move _all_negative_elements_to_end.java
class Solution {
    
    public void segregateElements(int arr[], int n)
    {
        // Your code goes here
        ArrayList<Integer> pos = new ArrayList<>();
        ArrayList<Integer> neg = new ArrayList<>();
        
        for(int i=0; i<n ;i++) {
            if(arr[i] >=0 ) {
                pos.add(arr[i]);
            }
            else {
                neg.add(arr[i]);
            }
        }
         
        int i=0;
        while(i<n) {
            for(int j=0; j<pos.size() ; j++) {
                arr[i]=pos.get(j);
                i++ ;
            }
            for(int k=0; k<neg.size() ; k++) {
                arr[i]=neg.get(k);
                i++ ;
            }
         }
    }
}

File: Solutions/Python/Move _all_negative_elements_to_end.py
class Solution:
    def segregateElements(self, arr, n):
        # Your code goes here
        l2=[]
        l3=[]
        for i in arr:
            if i>0:
                l2.append(i)
            if i<0:
                l3.append(i)
        arr.clear()
        l2.extend(l3)
        arr.extend(l2)
        return arr
